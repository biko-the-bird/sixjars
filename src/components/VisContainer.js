import React, {Component} from 'react';
import Visualization from './Visualization';
import Balance from './balance';
import {isMobile} from '../helpers';

class VisContainer extends Component {

    render() {
        console.log(isMobile());
        return (
            <div className="vis-container">
                <Balance/>
                {!isMobile() &&
                <Visualization/>
                }
            </div>
        )  
    }
}
export default VisContainer;