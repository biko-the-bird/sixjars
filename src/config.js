
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

//this is a configuration for enzyme that needs
//to be imported in each testing file
const config = configure({ adapter: new Adapter() });
export default config;