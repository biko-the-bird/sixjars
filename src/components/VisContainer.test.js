import React from 'react';
import {shallow} from 'enzyme';
import config from '../config';
import VisContainer from './VisContainer';


describe("vis-container", () => {
    const visContainer = shallow(<VisContainer/>);

    it('renders', () => {
        expect(visContainer).toMatchSnapshot();
    });
    it('contains a Connect(Balance) component', () => {
        expect(visContainer.find('Connect(Balance)').exists()).toBe(true);
    });
    it('contains a Connect(Visualization) component', () => {
        expect(visContainer.find('Connect(Visualization)').exists()).toBe(true);
    });
});

