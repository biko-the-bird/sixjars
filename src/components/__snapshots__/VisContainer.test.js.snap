// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vis-container renders 1`] = `ShallowWrapper {}`;
